pkgs/dart_services/lib/src/compiler.dart

```diff
@@ -224,8 +226,8 @@ class Compiler {
 
         final results = DDCCompilationResults(
           compiledJS: processedJs,
-          modulesBaseUrl: 'https://storage.googleapis.com/nnbd_artifacts'
-              '/${_sdk.versionFull}/',
+          modulesBaseUrl: 'https://storage.googleapis.com/$_storageBucket'
```

Comment:
Same as an earlier comment - do we need this to be configurable, or can we just always use storageBucket == `nnbd_artifacts`?

Reply:
See my answer above